include/etiss/SimpleMemSystem.h

```diff
@@ -89,8 +89,15 @@ class MemSegment
     const etiss::uint64 size_;
     access_t mode_;
 
+    /// @brief Constructor of Memory Segment
+    /// @param start_addr Start address of segment
+    /// @param size Size in bytes
+    /// @param mode Access Mode (R/W/X)
+    /// @param name Segment name
+    /// @param mem Pre-allocated Memory (not overwritten with initString)
+    /// @param initString String for initialization with imple_mem_system.memseg_initelement_ attr/ If not specified random value allocation
     MemSegment(etiss::uint64 start_addr, etiss::uint64 size, access_t mode, const std::string name,
-               etiss::uint8 *mem = nullptr, uint8_t initVal = 0)
+               etiss::uint8 *mem = nullptr, std::string initString = 0)
```

Comment:
std::string default argument should be `= ""` or removed since it now will be managed by etiss::initialize